jasmine/private/jasmine_test.bzl

```diff
@@ -0,0 +1,62 @@
+load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
+load("@bazel_skylib//lib:dicts.bzl", "dicts")   
+
+_attrs = dicts.add(js_binary_lib.attrs, {
+    "entry_point": attr.label(
+        mandatory = True,
+    ),
+    "config": attr.label(
+        doc = "jasmine config file. See: ",
+        allow_single_file = [".json", ".mjs", ".cjs", ".js"]
+    )
+})
+
+
+def _impl(ctx):
+
+    files = js_lib_helpers.gather_files_from_js_providers(
+        targets = ctx.attr.data,
+        include_transitive_sources = ctx.attr.include_transitive_sources,
+        include_declarations = ctx.attr.include_declarations,
+        include_npm_linked_packages = ctx.attr.include_npm_linked_packages,
+    )
+    files.extend(ctx.files.data)
+
+    fixed_args = [
```

Comment:
I feel like this should be an argument with this value as the default. Do we have other `*_test` rules with something such as a `srcs` or `tests` argument?

Reply:
I feel the same way.  Though https://github.com/bazelbuild/rules_nodejs/blob/8f2333cef700d2e94aead34cf710f07580896a68/packages/jasmine/jasmine_runner.js#L55 this was a constant in rules_nodejs and looks like nobody wanted to configure this.

Reply:
I guess it's easier to add it later then remove it later so stick with no argument for now?

Reply:
Exactly.